docs/v3/tutorials/alerts.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Send alerts to Slack on failure
```

Comment:
Here's an example of defining an automation in Python code: https://github.com/PrefectHQ/examples/blob/main/scripts/create-automation.py

Reply:
They can also be [defined with Terraform](https://registry.terraform.io/providers/PrefectHQ/prefect/latest/docs/resources/automation). We should mention that all those options are available.

Reply:
https://github.com/PrefectHQ/prefect/pull/16345/commits/9389e5c4f85fc5cfc4cdd8e926dccdc881cad9a6